.github/workflows/crucible-ci.yaml

```diff
@@ -22,19 +22,19 @@ jobs:
 
     steps:
     - name: checkout crucible-ci
-      uses: actions/checkout@v2
```

Comment:
I am not opposed to do this here, but this does not have anything with the change. A separate PR for this node.js bump would be more appropriate.

Reply:
They are separate commits, just part of the same PR.  It started spewing warnings about this while I was testing this PR and it was quite annoying so I went ahead and "fixed" it.